Header header
string node_name
uint64 cputime
uint64 diffcputime
uint64 diffwalltime
uint64 all_memory
uint64 resident_memory